docs/source/basics.rst

```diff
@@ -176,6 +176,25 @@ There is more information about creating stubs in the
 The following sections explain the kinds of type annotations you can use
 in your programs and stub files.
 
+Mypy will also look in some default install directory to try to find annotations.
+For example, for UNIX based, with python 3.4, it will try to find it in
+- ``${HOME}/.local/shared/typehints/python3.4/``
+- ``/usr/shared/typehints/python3.4/``
```

Comment:
I've opened [peps#67](https://github.com/python/peps/pull/67) about it, we can move the discussion there.